04_environment/infra.tex

```diff
@@ -0,0 +1,212 @@
+\cline{2-5}
+                                                                                      & islandDeplete      & \begin{tabular}[c]{@{}l@{}}Deducts an amount of~\\resources from each\\island that is proportional\\to remaining disaster \\effect~after CP mitigation\end{tabular} & \begin{tabular}[c]{@{}l@{}}- Effects experiened by\\each island after CP\\mitigation\end{tabular}                                                                                                                               & None                                                                                             \\
+\hline
+\end{tabular}
+\end{center}
+\caption{Main funcitions of Common Pool for Disaster Mitigation}
+\label{tab: Common Pool's Main Function}
+\end{table}
+
+
+This section details the implementation of Common Pool disaster mitigation. This includes a predetermined, constant \texttt{threshold} value that effectively represents a minimum required level of collective `insurance` against disasters. If a disaster occurs and the Common Pool resource level exceeds this threshold, the effects of the disaster are halved. If not, no such halving takes place and the full effect of the disaster is felt by the islands.
+
+An important input to the implementation is the magnitude of the disaster’s damage on each island. In Figure~\ref{fig:Common Pool infrastructure outcome}, this specific disaster’s magnitude was 3.458. However, the disaster’s effect on individual islands does not add up to 3.458 because the storm’s epicenter is located some distance away from each island. This aspect creates a more realistic scenario. From the individual damage experienced by each island, proportional effects on each island are computed. The summation of these proportional effects is equal to 100$\%$, which is the total effective damage.\newline
+
+If the common pool has enough resources to fully mitigate the disaster, the process is very easy as each island escapes unscathed. However, in the unfortunate event that the total disaster damage exceeds Common Pool resources, all such resources will be distributed to islands in proportion to the damage they experience (regardless of what their prior Common Pool contributions were). This is also visualised in Figure~\ref{fig:Common Pool infrastructure outcome}: Island 4 experiences the most damage from the storm and so, Island 4 receives the maximum Common Pool resource allocation of 252.47 resources (“common pool mitigated” column). The updated damage value shows the leftover damage of the disaster that the common pool was not able to mitigate. Left over damage serves to deplete an island's private pool of resources where applicable. If there is enough resources in the common pool to totally mitigate the disaster, the update damage values will be zero.
```

Comment:
It should be explicitly stated that the disaster first takes away from the common pool and then affect the islands. 
```suggestion
When a disaster strikes the effect of the disaster on each islands is summed compared to the resources in the common pool. If the common pool has more resources than the total effect of the disaster it can fully mitigate the disaster meaning the islands take not damage. However, in the unfortunate event that the total disaster damage exceeds Common Pool resources, the common is first depleted and the total disaster effect is also reduced by the same amount and then the damage is applied to islands in proportion to the original damage they would have experienced (regardless of what their prior Common Pool contributions were). This is also visualised in Figure~\ref{fig:Common Pool infrastructure outcome}: Island 4 experiences the most damage from the storm and so, Island 4 receives the maximum Common Pool resource allocation of 252.47 resources (“common pool mitigated” column). The updated damage value shows the leftover damage of the disaster that the common pool was not able to mitigate. Left over damage serves to deplete an island's private pool of resources where applicable. If there is enough resources in the common pool to totally mitigate the disaster, the update damage values will be zero.
```

Apologies for the long suggestion

Reply:
Thank you : )